extensions/amp-social-share/0.2/social-share.js

```diff
@@ -30,63 +32,196 @@ const DEFAULT_HEIGHT = 44;
  * @param {!JsonObject} props
  * @return {PreactDef.Renderable}
  */
-export function SocialShare(props) {
+export function SocialShare2(props) {
```

Comment:
"2"?

Reply:
I changed the name, because I was concerned that I would break the amp component.  The old version of the preact component has different behavior, and the amp component relies on the old version.  Should I remove the 2?

Reply:
Yes. Let's break.